pkg/util/selector.go

```diff
@@ -1,14 +1,24 @@
 package util
 
 import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
 
 	clusterv1alpha1 "github.com/karmada-io/karmada/pkg/apis/cluster/v1alpha1"
 	"github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
 )
 
+const (
+	providerField = "provider"
+	regionField   = "region"
+	zoneField     = "zone"
```

Comment:
make them as common variables? so the [case](https://github.com/karmada-io/karmada/pull/357/files#diff-4fe5d873ba71c3512cafdeeb480dce209710dd2b5ed84d900522e1bc07aa7f90R86) can use them.